tests/io/test_bedio.py

```diff
@@ -0,0 +1,100 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
+# and any modifications thereto.  Any use, reproduction, disclosure or
+# distribution of this software and related documentation without an express
+# license agreement from NVIDIA CORPORATION is strictly prohibited.
+#
+
+from atacworks.io import bedio
+
+import pandas as pd
+import pytest
+import os
+
+
+@pytest.mark.cpu
+def test_read_intervals(tmpdir):
+    """ Test read intervals."""
```

Comment:
general comment for all the tests - it would be good to have docstrings which explain a brief of what the test is doing

e.g.
```
This test checks for correctness of read_intervals API by writing intervals to a bed file and then reading in the same file and checking for equivalance.
```

Reply:
done.